### DISCUSSION TITLE
take to the same page when pressing Login button  + not notify or pop up if the pass or email incorrect

### DISCUSSION THREAD

**Discussion by BIOHS** [community/community#133547] [Programming Help]
### Body

When I login it takes me to the same page when I press Login button 
+ if I write the password or email incorrect its login but also take me to the same page
how can I solve these 2 issues?

this is the codes :
https://github.com/BIOHS/Demo

My idea of this is making login page, I can contact the people that want to subscribe with the website, and write their users in a spread sheet so the page can accept logging them in to see courses (I'm not including yet the courses, I'm still working in this project) and giving them the pass to log in.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by NicholasGannon** [comment]
- It appears that the **href** value for that Login button is set to index.html: ```<a href="index.html">Log In</a>```
- I assume you'll have to create that **other page** you want, and route the href to it.

**Comment by Azymack** [comment]
It's not a good practice to implement auth functionality on front-end side as it can be easily hacked even on browser.
You ain't gonna have any way to validate correctly whether user's logged in or not with setting up back-end server.
